Model/articles.model.js

```diff
@@ -4,8 +4,29 @@ exports.selectArticleById = (article_id) => {
 
     return db.query('SELECT * FROM articles WHERE article_id = $1', [article_id]).then(results => {
         if(results.rows.length === 0){
-                return Promise.reject({status: 404, msg: 'article doesnt exist'})
+            return Promise.reject({status: 404, msg: 'article doesnt exist'})
         }
         return results.rows[0]
     })
+}
+
+exports.updateArticleById = (article_id, inc_votes) => {
+    
+    return db.query('SELECT * FROM articles WHERE article_id = $1', [article_id]).then(results => {
+
+        if(results.rows.length === 0){
+            return Promise.reject({status: 404, msg: 'article doesnt exist'})
+        } 
+
+        const checkVotes = Math.sqrt(Math.pow(inc_votes, 2))
+        if(checkVotes > results.rows[0].votes) {
+            return Promise.reject({status: 400, msg: 'bad request'})
+        }
+
+        const value = results.rows[0].votes + inc_votes
+        
```

Comment:
Hey! What is the purpose of the checkVotes? It seems a little complex.

I would also say the need for two requests is unnecessary here as we could just query using the update method. Have a look at how you could include the votes to increment by within an operation in your UPDATE query